templates/grommet-v2-template/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "grommet-v2-template",
+  "version": "1.0.0",
+  "description": "template for grommet v2. used as a validation test for circleci post publish",
+  "keywords": [],
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "grommet": "^2.6.5",
```

Comment:
This seems wrong to me, it will not test the actual release rather than just the 2.6.5
Should probably be latest here?

Reply:
my thought was that the new release was already published, hence `grommet` will pick the latest (and not `2.6.5`), this is why I added the ` ^ ` operator so it will pick the most recent release. 
let me know if it makes sense.

Reply:
It will not work on `3.0.0` though. i am also not sure actually if it will get the one we want even if it is latest due to cache. We ll have to see :)

Reply:
I don't have a strong opinion here, I'm ok changing it to `"grommet": "latest"` if you think it will work better. let me know.